
By default synapse and ulauncher use the same hotkey to show the launcher window.
To avoid a collision, set the hotkey differently for ulauncher. - Didier Spaier

diff -ru ulauncher/ulauncher/utils/Settings.py ulauncher.new/ulauncher/utils/Settings.py
--- ulauncher/ulauncher/utils/Settings.py	2023-11-13 00:18:08.000000000 +0100
+++ ulauncher.new/ulauncher/utils/Settings.py	2024-03-17 22:52:48.232572382 +0100
@@ -23,7 +23,7 @@
     "hotkey-show-app": (str,  # type
                         "Hotkey: Show ulauncher window",  # nick name
                         None,  # description
-                        "<Primary>space",  # default
+                        "<Primary>Super_L",  # default
                         GObject.ParamFlags.READWRITE),  # flags
     "show-indicator-icon": (bool,
                             "Show app indicator",
